src/main/java/mate.academy/Main.java

```diff
@@ -6,11 +6,11 @@
 import mate.academy.service.ProductService;
 
 public class Main {
-
     public static void main(String[] args) {
         // Please test your Injector here. Feel free to push this class as a part of your solution
         Injector injector = Injector.getInjector();
-        ProductService productService = null;
+        ProductService productService = (ProductService) injector.getInstance(
+                ProductService.class);
```

Comment:
Ensure that the Injector class is correctly implemented and configured to return an instance of ProductService. If the Injector is not properly set up, this line may cause a runtime error.